shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -600,6 +567,71 @@ - (void)shutDownEngine {
   _engine = nullptr;
 }
 
+- (FlutterCompositor*)createFlutterCompositor {
```

Comment:
In macOS embedding we declare the private methods in the top and add comments.

Reply:
Done